### ISSUE TITLE
Prefer GET to POST

### ISSUE THREAD

**Issue by GlazerMann** [ms609/citation-bot#4480]
proxy cachable

**Comment by GlazerMann** [comment]
Some APIs do not:
support GET
do not work with funky characters
do not work with large requests

So, test carefully

**Comment by maximmasiutin** [comment]
let me add a condition for funky characters, it is probably the reason for "11β-Hydroxyprogesterone"

**Comment by maximmasiutin** [comment]
> let me add a condition for funky characters, it is probably the reason for "11β-Hydroxyprogesterone"

Sorry, didn't manage to implement it properly. Let me cancel my implementation attempt of changes of GET to POST for now, but let's keep in mind for the future to prefer GET wherever possible. By the way, since you changed all calls to curl, the use of citation bot via proxy works very well, with the only exception that it cannot cache requests made via POST; still even when I tried the code which didn't work well with funky characters, it worked very well if you consider the number of request saved by proxy (I used latest Squid for Linux).
